system-tests/jobs/test-jobs.js

```diff
@@ -7,7 +7,7 @@ describe("Jobs", function() {
     });
   });
 
-  it("Create a simple job", function() {
+  it("should create a simple job", function() {
```

Comment:
I'm with @weblancaster 

I think we could learn from ruby http://www.betterspecs.org/ I also tend to write what a system _does_ instead of what it _should_ do. So I'd write `it("creates a simple job")` let's team sync on this one.